src/js/components/month.jsx

```diff
@@ -7,11 +7,12 @@ export default class Month extends React.Component {
   constructor(props) {
     super(props);
 
-    const {
-      date,
-      weeks,
-    } = this.props;
+    this.state = {
+      currentWeekIndex: this._getStartingWeek(props.date, props.weeks),
```

Comment:
Hmm while **date** isn't exactly a bad way to refer to the current date, I think It'd be wiser to use **currentDate**, since this leaves less room for mind-guessing what the hell is that date.